tidb-cloud/connect-via-standard-connection-serverless.md

```diff
@@ -32,6 +32,16 @@ To connect to a TiDB Serverless cluster via public endpoint, take the following
     >
     > When you connect to a TiDB Serverless cluster, you must include the prefix for your cluster in the user name and wrap the name with quotation marks. For more information, see [User name prefix](/tidb-cloud/select-cluster-tier.md#user-name-prefix).
 
+## Obtain TiDB Serverless Connection Parameters
+
+In the connection dialog, you can obtaining the connection parameters follow these instractions. If you don't know how to enter the connection dialog, please go back to the [top of this document](#connect-to-tidb-serverless-via-public-endpoint) to get more information.
```

Comment:
Better to add a screenshot.

Reply:
@Oreoxmt, PTAL. @Mini256 and I have the same opinion.

Reply:
@Mini256 We minimize screenshots of the TiDB Cloud console in our documentation for a few reasons:

- The Connect dialog is designed to be user-friendly and self-explanatory, so textual instructions are sufficient for users to follow and connect to their clusters without the need of a visual reference.
- The interface of TiDB Cloud console changes frequently, so keeping screenshots updated is time-consuming and outdated images can lead to confusion.

Reply:
1. Please don't assume that the existing design is good enough, especially if you are already familiar with the product. Let's take a look at what similar products that have been well-received by developers are doing. Not only do they add screenshots, but they even add videos to help users get through quick start faster!

    - https://planetscale.com/docs/tutorials/connect-any-application
    - https://planetscale.com/docs/tutorials/connect-mysql-gui
    - https://planetscale.com/docs/tutorials/planetscale-quick-start-guide
    - https://docs.yugabyte.com/preview/develop/build-apps/cloud-add-ip/
2. I know you will be thinking about maintenance costs, but that **SHOULD NOT** be the reason that keeps preventing us from improving the developer experience on the documentation. Also, if there is a confusion due to outdated images, I believe we need to go through and check if the text descriptions need to be updated.

Reply:
@Icemap @Mini256 Totally understood. What you said is ideal and is what to expect. 

If every developer or PM can have the sense to update the screenshot once it's changed in UI or at least let us know and request us to help update, that would be great. If NOT, outdated screenshot with correct text can be more confusing to users that just correct and clear text. Let's work together to improve the status quo but it takes time. And bandwidth is what I must consider.

If a UI element is so hard to find, this is a valuable input for the designer to improve it, which is an upstream solution.

Reply:
@shizn, what do you think of this question, do you have any suggestions?